app/views/mainstream_browse_pages/edit.html.erb

```diff
@@ -1,18 +1,29 @@
-<%= tag_header @browse_page, 'Edit' %>
+<%= render 'breadcrumbs', {
+  links: [
+    {
+      text: 'Mainstream browse pages',
+      href: mainstream_browse_pages_url
+    },
+    *([
+      text: @browse_page.parent[:title],
+      href: mainstream_browse_page_url(@browse_page.parent)
+    ] if @browse_page.parent.present? ),
+    {
+      text: @browse_page[:title],
+      href: mainstream_browse_page_url(@browse_page)
+    },
+    {
+      text: 'Edit',
+    }
+  ]
+} %>
 
-<%= form_for @browse_page do |f| %>
-  <%= f.text_field :slug, :disabled => true %>
-  <%= f.text_field :title %>
-  <%= f.text_field :description %>
+<% content_for :title, govuk_tag_heading(tag: @browse_page, prepend: 'Edit') %>
```

Comment:
Is this header supposed to include the tag?

Currently it causes a visual jump when moving from `show.html.erb` to `edit.html.erb`.

| Show page | Edit page |
| --- | --- |
| <img width="314" alt="show" src="https://user-images.githubusercontent.com/7735945/161104393-de6cce59-aa76-436a-99ab-db9b739733b5.png"> | <img width="367" alt="edit" src="https://user-images.githubusercontent.com/7735945/161103568-4e1d6a51-d0d0-4245-8898-91b7ff0212a1.png"> |

Reply:
This is how it's currently styled, we decided to stick to the current styling until we have a designer go through each bit and revise it. (I agree with what you're saying, not very pleasing for users)